client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -46,18 +46,27 @@ const ManageAccountView = () => {
       bodyClassName="has-loaded-account"
       introMessageName="ACCOUNT"
       header="Manage Account"
-      messages={
+      messages={[
+        !isLoading && (errors.data || errors.fields) && (
+          <SectionMessage type="error">
+            An error occurred loading your profile information.
```

Comment:
**Request from Design.** Please change the wording to reflect the scope of the error (which Design now understands to encompass more data[^1] than originally thought).

```suggestion
            An error occurred loading your account information.
```

<details>
<summary>Details</summary>

See https://github.com/TACC/Core-Portal/pull/543#discussion_r747033432.

</details>

[^1]: Design and I understand the error would appear for an error loading __any__ data in the section, not just the "profile" (which design considers to be "Required information" and "Optional information").